lib/win32/windows/time_calc_helper.rb

```diff
@@ -0,0 +1,116 @@
+module Windows
+  module TimeCalcHelper
+
+    DAY_OF_MONTH = [0, 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
```

Comment:
We don't use 0th index, but the structure of this array is maintained in a way that it returns actual no of days in a month (directly with month number), without any manipulation
```
eg, JAN(month No: 1) => DAY_OF_MONTH[1] => 31
Usage: @#L8:
(month == 2 && is_leap_year?(year)) ? 29 : DAY_OF_MONTH[month]
```
Also, I've kept this' 0' purposely. Keeping a 'nil' or 'a blank string' at first place _could be prone to exceptions_ in calculations and hence it may _require a conversion_ (to_i), which is an overhead.(Please let me know your views)

Have updated PR with comment in code explaining its significance.
Thank you